ext/liquid_c/tokenizer.c

```diff
@@ -66,20 +67,28 @@ void tokenizer_next(tokenizer_t *tokenizer, token_t *token)
 
     token->str = cursor;
     token->type = TOKEN_RAW;
+    token->lstrip = 0;
+    token->rstrip = 0;
 
     while (cursor < last) {
         if (*cursor++ != '{')
             continue;
 
         char c = *cursor++;
+        char w = *cursor++;
```

Comment:
This can read past the end of the string.  E.g. tokenizing `"{{"` results in a token with `"{{\u0000"`.  The `while (cursor < last)` check only allows us to read two characters before having to do another bounds check to see if we can read another byte.